client/src/App.jsx

```diff
@@ -36,6 +36,16 @@ export default class App extends Component {
     this.toggleDemo = this.toggleDemo.bind(this);
   }
 
+  getUserData(userEmail) {
```

Comment:
We can go ahead and delete these GET and POST methods at the app level, they exclusively live and are called from the utils folder now.